lm_eval/__main__.py

```diff
@@ -167,7 +167,6 @@ def cli_evaluate(args: Union[argparse.Namespace, None] = None) -> None:
     if (args.log_samples or args.predict_only) and not args.output_path:
         assert args.output_path, "Specify --output_path"
 
-    initialize_tasks(args.verbosity)
     task_manager = TaskManager(args.verbosity, include_path=args.include_path)
```

Comment:
Is it okay for us to remove this now instead of the next release?

Reply:
Sure can add it back!